data-service/src/main/java/uk/gov/gchq/palisade/service/data/web/DataController.java

```diff
@@ -91,11 +91,16 @@ public DataController(final AuditableDataService auditableDataService, final Aud
                 AuditableDataResponse auditableDataResponse = auditableDataService.read(auditableAuthorisedDataRequest, outputStream).join();
                 auditMessageService.auditMessage(TokenMessagePair.Builder.create()
                         .withToken(dataRequest.getToken())
+                        //send a message to the audit service of successfully processed request
                         .withAuditMessage(auditableDataResponse.getAuditSuccessMessage()));
 
                 Optional.ofNullable(auditableDataResponse.getAuditErrorMessage())
-                        .ifPresent(errorMessage -> auditMessageService.auditMessage(TokenMessagePair.Builder.create()
-                                .withToken(dataRequest.getToken()).withAuditMessage(errorMessage)));
+                        .ifPresent((AuditErrorMessage errorMessage) -> {
+                            //send a message to the audit service of an error occurred in processing a request
+                            LOGGER.error("Error occurred processing the read  error message was {}", errorMessage);
```

Comment:
```suggestion
                            LOGGER.error("Error occurred processing the read, error message was: {}", errorMessage);
```
Again, maybe the "error message was" could be removed?

Reply:
Removed the last few words and added the ":".